library_generation/new_client/templates/owlbot.py.j2

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import synthtool as s
+{% if should_include_templates %}from synthtool.languages import java{% endif %}
+
+
+for library in s.get_staging_dirs():
+    # put any special-case replacements here
+    s.move(library)
+
+s.remove_staging_dirs()
+{% if should_include_templates %}java.common_templates(monorepo=True, {% if template_excludes %}excludes={{ template_excludes }}{% endif %}){% endif %}
```

Comment:
That `template_excludes` variable always comes as an empty array when called from `new-client.py`, so maybe it's better to just hardcode it here. I guess this happens because this template file was recycled from somewhere else

Reply:
I think maybe it's good to keep it as-is because we don't want to change the logic in this PR.

We can make refactor in further PRs.